### DISCUSSION TITLE
src中存在软链文件夹，max dev不报错，max build却编译报错

### DISCUSSION THREAD

**Discussion by jiadongy** [umijs/umi#11128] [Q&A]
Win10环境，src中存在一个软链文件夹，max dev不报错，max build却编译报错

fatal - Error: ERROR in ../../appx/base-web/components/Avatar.tsx 6:0-51
Module not found: Error: Can't resolve '@ant-design/icons' in 'C:\Users\my\Projects\appx\base-web\components'
 @ ../../appx/base-web/pages/user/settings/components/BaseTab.tsx 4:0-60 70:34-46
 @ ../../appx/base-web/pages/user/settings/index.tsx 4:0-43 21:34-41
 @ ./src/.umi-production/core/route.tsx
 @ ./src/.umi-production/umi.ts 11:0-41 28:17-26

似乎是route.tsx中使用了软链的真实路径

请问该怎么处理

**Comment by fz6m** [comment]
`@ant-design/icons` 这个依赖是幽灵依赖吧，装一下这个依赖。
